src/transformers/configuration_t5.py

```diff
@@ -30,6 +30,9 @@
     "t5-11b": "https://s3.amazonaws.com/models.huggingface.co/bert/t5-11b-config.json",
 }
 
+T5_LANGUAGES_FROM = ["English"]
+T5_LANGUAGES_TO = ["French", "German", "Romanian"]
+
```

Comment:
I put the T5 allowed language constants here - what do you think @thomwolf ?

Reply:
I would probably like to have them inside the config dict if we can do it (see my discussion on #3413) regarding task-specific attributes in configuration files.

Also, I think we can probably just replace this with 3 tasks here for now: `translate_en_to_fr`, `translate_en_to_de` and `translate_en_to_ro`, it will be easier to define language-specific generation HPs.

Open to other options though but being able to have language pair specific HP is probably nice and defining them as tasks looks like a simple first way to do that.